python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py

```diff
@@ -439,18 +448,30 @@ def __call__(self, data: KeyedData, **kwargs) -> KeyedData:
         ra: Vector = data[self.colRa]  # type: ignore
         dec: Vector = data[self.colDec]  # type: ignore
 
-        treecorrKwargs = self.treecorr.toDict()
+        treecorr_config_dict = self.treecorr.toDict()
+
+        # Swap rng_seed with an rng instance in treecorr config.
+        rng = np.random.RandomState(treecorr_config_dict.pop("rng_seed"))
+        treecorr_config_dict["rng"] = rng
```

Comment:
The `rng_seed` remains the same for all the runs for now, but a new `rng` is generated for each `tract, patch, band` combo. I suppose the actual patches themselves depends on the data (since it uses kmeans) and it's not the case where we get the same patch every single run, right?